newIDE/app/src/EventsFunctionsExtensionEditor/OptionsEditorDialog/ExtensionDependenciesEditor.js

```diff
@@ -0,0 +1,168 @@
+            {mapVector<gdDependencyMetadata, TableRow>(
+              eventsFunctionsExtension.getAllDependencies(),
+              (dependency, index) => (
+                <TableRow key={dependency.getName()}>
+                  <TableCell>
+                    <SemiControlledTextField
+                      commitOnBlur
+                      value={dependency.getName()}
+                      onChange={newName => {
+                        if (checkNameExists(newName, deps))
+                          Window.showMessageBox(
+                            `This name is already in use! Please use a unique name.`,
+                            'error'
+                          );
+                        else
+                          eventsFunctionsExtension
+                            .getAllDependencies()
+                            .at(index)
+                            .setName(newName) && forceUpdate();
+                      }}
+                    />
+                  </TableCell>
+                  <TableCell>
+                    <SemiControlledTextField
+                      commitOnBlur
+                      value={dependency.getExportName()}
+                      onChange={newExportName =>
+                        dependency.setExportName(newExportName) && forceUpdate()
+                      }
+                    />
+                  </TableCell>
+                  <TableCell>
+                    <SemiControlledTextField
+                      commitOnBlur
+                      value={dependency.getVersion()}
+                      onChange={newVersion =>
+                        dependency.setVersion(newVersion) && forceUpdate()
+                      }
+                    />
+                  </TableCell>
+                  <TableCell>
+                    <SelectField
+                      value={dependency.getDependencyType()}
+                      onChange={(_, __, newType) =>
+                        dependency.setDependencyType(newType) && forceUpdate()
+                      }
+                    >
+                      <SelectOption value="npm" primaryText={t`NPM`} />
+                      <SelectOption value="cordova" primaryText={t`Cordova`} />
+                    </SelectField>
+                  </TableCell>
+                  <TableCell>
+                    <IconButton
+                      tooltip={t`Remove`}
+                      onClick={() => {
+                        eventsFunctionsExtension.removeDependencyAt(index);
+                        forceUpdate();
+                      }}
+                    >
+                      <Delete />
+                    </IconButton>
+                  </TableCell>
+                </TableRow>
+              )
+            )}
+          </TableBody>
+        </Table>
+        <FlatButton
+          icon={<Add />}
+          label={<Trans>Add</Trans>}
+          onClick={addDependency}
+        />
+      </TableContainer>
+      <br />
```

Comment:
Don't use br :) Use Line/Column/Spacer, as they ensure a consistent spacing across the app.